traffic_ops/traffic_ops_golang/deliveryservice/deliveryservice_server_capabilities.go

```diff
@@ -0,0 +1,196 @@
+package deliveryservice
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/apache/trafficcontrol/lib/go-tc"
+	"github.com/apache/trafficcontrol/lib/go-tc/tovalidate"
+	"github.com/apache/trafficcontrol/lib/go-util"
+	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/api"
+	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/dbhelpers"
+	validation "github.com/go-ozzo/ozzo-validation"
+)
+
+const (
+	deliveryServiceQueryParam  = "deliveryServiceID"
+	serverCapabilityQueryParam = "serverCapability"
+	xmlIDQueryParam            = "xmlID"
+)
+
+// ServerCapability is a type alias to define functions on
+type ServerCapability struct {
+	api.APIInfoImpl `json:"-"`
+	tc.DeliveryServiceServerCapability
+}
```

Comment:
What does "type alias" mean in this context? Go has a specific meaning for a type alias in the spec (https://golang.org/ref/spec#Type_declarations), but this code is not declaring that kind of type alias. Consider different language in the comment here to avoid confusion.